### ISSUE TITLE
Petra Vendor unavailable

### ISSUE THREAD

**Issue by ttgmpsn** [Bungie-net/api#655]
Hey again, could Petra Venj in the Dreaming City (Vendor Hash: 1454616762) please be enabled once the character you query it for reached the location? Right now the API just returns DestinyVendorNotFound, despite the character having access to it.

**Comment by vthornheart-bng** [comment]
Just to check - has the character in question completed the Talisman quest?

**Comment by ttgmpsn** [comment]
Yes, no open quests for the character in question. Can do the bounties in-game just fine.

**Comment by vthornheart-bng** [comment]
There does seem to be something unusual happening there, but I haven't had the time to do a real root cause analysis yet.  I'll file a real bug for this tomorrow!

**Comment by ttgmpsn** [comment]
Looks like you made #704 for better awareness, guess this can be closed as dupe :)
